other/add-default-securitycontext/add-default-securitycontext.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       patchStrategicMerge:
         spec:
           securityContext:
+            +(runAsNonRoot): true
```

Comment:
Have you tested this that it works properly without setting to `"true"`? The spirit of this is good, just need to make sure it works as written (boolean) and doesn't require string.

Reply:
I've updated it as "true" 🙋🏻‍♂️

Reply:
So does it work?

Reply:
It should be a boolean btw, I got an error when I change it as string

> kyverno-568449c9d6-fdptz kyverno E0131 16:19:10.359522 1 mutation.go:86] WebhookServer "msg"="mutate error" "error"="failed to validate resource mutated by policy add-default-securitycontext: ValidationError(io.k8s.api.core.v
1.Pod.spec.securityContext.runAsNonRoot): invalid type for io.k8s.api.core.v1.PodSecurityContext.runAsNonRoot: got "string", expected "boolean"" "action"="mutate" "gvk"="/v1, Kind=Pod" "operation"="CREATE" "resource"="default/Pod/"

Reply:
Ok